include/cpuinfo.h

```diff
@@ -499,6 +499,9 @@ enum cpuinfo_uarch {
 	/** Applied Micro X-Gene. */
 	cpuinfo_uarch_xgene = 0x00B00100,
 
+	/** Huawei Kunpeng Serial CPU. */
+	cpuinfo_uarch_tsv100 = 0x00C00100,
```

Comment:
According to wikichip [1], the microarchitecture is called "TaiSHan v110". Please use the full name rather than an abbreviation to be consistent with other values in this enum (e.g. we use `cpuinfo_uarch_haswell` rather than `cpuinfo_uarch_hsw`)

[1] https://en.wikichip.org/wiki/hisilicon/kunpeng/920-6426

Reply:
remove it